### ISSUE TITLE
Storybook 3.0 support

### ISSUE THREAD

**Issue by peggyrayzis** [storybooks/addon-backgrounds#30]
Hey there! Love this Storybook addon. Unfortunately, I'm running into some problems when upgrading to Storybook 3.0.

Here are the errors from my Webpack build:
```
ERROR in ./~/react-storybook-addon-backgrounds/dist/register.js
Module not found: Error: Can't resolve '@kadira/storybook-addons' in '/Users/mrayzis/projects/mls/ui-packages/packages/react-components/node_modules/react-storybook-addon-backgrounds/dist'
 @ ./~/react-storybook-addon-backgrounds/dist/register.js 3:25-60
 @ ./~/react-storybook-addon-backgrounds/register.js
 @ ./.storybook/addons.js
 @ multi ./.storybook/addons.js ./~/@storybook/react/dist/server/config/polyfills.js ./~/@storybook/react/dist/client/manager/index.js

ERROR in ./~/react-storybook-addon-backgrounds/dist/index.js
Module not found: Error: Can't resolve '@kadira/storybook-addons' in '/Users/mrayzis/projects/mls/ui-packages/packages/react-components/node_modules/react-storybook-addon-backgrounds/dist'
 @ ./~/react-storybook-addon-backgrounds/dist/index.js 8:25-60
 @ ./stories/match-extensions.js
 @ ./stories \.js$
 @ ./.storybook/config.js
 @ multi ./~/@storybook/react/dist/server/config/polyfills.js ./~/@storybook/react/dist/server/config/globals.js (webpack)-hot-middleware/client.js?reload=true ./.storybook/config.js
```

To fix it, I think you just have to rename the packages from `@kadira` to their new namespace `@storybook` as outlined in the [migration guide](https://github.com/storybooks/storybook/blob/master/MIGRATION.md#deprecated-embedded-addons). Thanks!

**Comment by shilman** [comment]
Thanks @peggyrayzis and sorry for the hassle. We just took over this addon from Newspring, but have not yet merged it into the new 3.0 world. We'll get on that soon! (cc @ndelangen)

**Comment by peggyrayzis** [comment]
No prob @shilman! The upgrade experience was painless otherwise, even with a custom Webpack config. Big thanks to the team for the codemod & clear migration guide. 👍

**Comment by jbaxleyiii** [comment]
@peggyrayzis yall used this plugin?! small world! 👋

Thanks again @ndelangen for moving this into core!

**Comment by peggyrayzis** [comment]
Oh heyyyy! 🙋 @jbaxleyiii did you write this?! I can't live without this plugin, it's one of my favs.

**Comment by jbaxleyiii** [comment]
@peggyrayzis yep! Sorry it was poorly maintained haha

**Comment by ndelangen** [comment]
@jbaxleyiii could you give me admin rights on this package on npm?

Right now I can't just simply merge this repo in because it's typescript.
Adding typescript to the core repo is not something I want to do for various reasons.

If someone is interested in converting this to plain JS? That'd help speed this along greatly

**Comment by soosap** [comment]
Has anyone figured out why this is not working w/ storybook v3. I wonder if there is a temporary quick fix I could resort to get this running again.

**Comment by ndelangen** [comment]
Is anyone interested in making a PR to add support?

**Comment by abstracthat** [comment]
https://github.com/storybooks/react-storybook-addon-backgrounds/pull/31

Hopefully merged soon?

**Comment by ndelangen** [comment]
I didn't get a notification for that PR! I'm on it!

**Comment by ndelangen** [comment]
I merged it, and will continue a bit of work to rename the package and do a release!

**Comment by ndelangen** [comment]
I got the master green and released: `@storybook/addon-backgrounds` `3.0.0`
